packages/core/src/util/normalizeConfig.js

```diff
@@ -9,11 +9,58 @@
 
 const supportedTracingVersion = require('../tracing/supportedVersion');
 
+/**
+ * @typedef {Object} HTTPTracingOption
+ * @property {Array<*>} [extraHttpHeadersToCapture]
+ */
+
+/**
+ * @typedef {Object} InstanaTracingOption
+ * @property {boolean} [enabled]
+ * @property {boolean} [automaticTracingEnabled]
+ * @property {number} [forceTransmissionStartingAt]
+ * @property {number} [maxBufferedSpans]
+ * @property {number} [transmissionDelay]
+ * @property {number} [stackTraceLength]
+ * @property {HTTPTracingOption} [http]
+ * @property {Array<*>} [disabledTracers]
+ * @property {boolean} [spanBatchingEnabled]
+ * @property {boolean} [disableAutomaticTracing]
+ * @property {boolean} [disableW3cTraceCorrelation]
```

Comment:
These typedefs are pretty great! Now you can immediately see what config values we support and it is even type checked! 👍 

I wonder if we could export them as an external type definition so customers that use type script could use them. But that is something for later.

Reply:
We can make use of TS automatic declaration file generation I suppose. This should be enough to provide proper typing for customers who actually use Typescript. I can play around that in the next type checking round